contracts/libraries/PermissionLibrary.sol

```diff
@@ -25,13 +24,10 @@ library PermissionLibrary {
         bytes32 FILE_READ_ROLE;
         bytes32 FILE_WRITE_ROLE;  
```

Comment:
Good question! Usually ACL roles are defined in the main smart contract itself but these roles are strictly used with the ObjectACL, and only by the PermissionLibrary. I think it would indeed make more sense to have them in the library.

Reply:
Done! :) I've also moved `DATASTORE_GROUP` to the GroupLibrary